### ISSUE TITLE
Question about the adv loss

### ISSUE THREAD

**Issue by ShawnPi233** [YoungSeng/Interspeech2022#24]
Thanks for your great works. I have some doubt in the computation of adversarial loss in `solver.py:` 
`  spk_loss = self.advloss(spk_pred.to(self.device), emb_org)
    content_adv_loss = self.advloss(content_pred, emb_org)
    g_loss += self.lambda_cd * spk_loss + self.lambda_cd * content_adv_loss`. Since the `emb_org` refers to the speaker embedding from one-hot projection, `spk_loss` is easy to understand. But `content_adv_loss`  should be maximized to fool the classifier so as to make sure the `content_pred` not contain speaker information. So I think it should be like this: `content_adv_loss = -self.advloss(content_pred, emb_org)` or the total loss should subtract the original `content_adv_loss` instead of adding it. Looking forward to your earliest reply at your convenience!
